cmd/microcloud/ask.go

```diff
@@ -1412,8 +1421,14 @@ func (c *initConfig) askCephNetwork(sh *service.Handler) error {
 			return err
 		}
 
+		iface, err := lxd.FindInterfaceForSubnet(internalCephSubnet)
+		if err != nil {
+			return err
```

Comment:
Can they return `fmt.Errorf("Failed to find interface for subnet %q: %w", net, err)`? Then the error messages within `FindInterfaceForSubnet` can be minimal (see comment from before).

Reply:
fixed